lib/ocpp/v201/charge_point.cpp

```diff
@@ -209,8 +209,8 @@ void ChargePoint::start(BootReasonEnum bootreason) {
     // get transaction messages from db (if there are any) so they can be sent again.
     this->message_queue->get_persisted_messages_from_db();
     this->boot_notification_req(bootreason);
-    // K01.27 - call load_charging_profiles when system boots
-    this->load_charging_profiles();
+    // K01.27 - call clear_invalid_charging_profiles when system boots
+    this->clear_invalid_charging_profiles();
```

Comment:
I'm confused `K01.FR.27` is "ChargingProfiles set via SetChargingProfileRequest
SHALL be persistent across reboots/power cycles." Why is it being changed to clearing them?

Reply:
Thank you @folkengine! I removed "K01.27 from the comment on line 212 and adjusted the description of the changes of this PR to note that the Database handler is handling the fulfillment of FR K01.27.